test/e2e/setup/gcp/Makefile

```
@@ -0,0 +1,36 @@
+	${e2e_dir}/pkg -- \
+	-api-server="http://127.0.0.1:8000" \
+	-grpc-server="127.0.0.1:8090" \
+	-server-kubeconfig=$(KUBECONFIG) \
+	-agent-kubeconfig=$(KUBECONFIG) \
+	-consumer-name=$(shell cat ${PWD}/_output/consumer_id) \
+	-dump-logs=false
+.PHONY: gcp/e2e-test
+
+gcp/teardown:
+	./setup/teardown.sh
+.PHONY: gcp/teardown
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)